components/layout_2020/flow/inline/mod.rs

```diff
@@ -92,6 +92,7 @@ use serde::Serialize;
 use servo_arc::Arc;
 use style::computed_values::text_wrap_mode::T as TextWrapMode;
 use style::computed_values::vertical_align::T as VerticalAlign;
+use style::computed_values::direction::T as ComputedCSSDircetion;
```

Comment:
You have a typo here. `Dircetion` instead of `Direction`

Reply:
TY. Will fix

Reply:
Fixed. Want to ask what is the policy here. I like to solve small typos by git amend and rewriting the original commit with forced push. Is it okay here? Or would you prefer to have separate commit?

Reply:
Either is fine! CI deals with force pushes to branches, and we squash merge so extra commits disappear anyway.